### ISSUE TITLE
An update was released for New Leaf!

### ISSUE THREAD

**Issue by nanashi89** [kwsch/NLSE#17]
How about updating your NLSE to include maxed out M.E.O.W. coupons? Plz?

Thx in advance!

**Comment by Will-The-Lion** [comment]
This new update looks like both the online and offline editors are totally incompatible with the new update. The save file format has changed some I'd guess. The name of the files at least is different. **_garden.dat_** has changed to _**garden_plus.dat**_ and there is a new file called _**updated.dat**_ that wasn't there previously.

I'm hoping for some new update to either editor to help get modding the game up and running again. 

I previously used it to rearrange my town and now with the new update it fails to assign a location for the campground exit and so I can't access the biggest portion of the new update. I'm hoping for a fix for that most of all!

**Comment by Ajadaz** [comment]
Yes I really hope to get it updated. I wanted to change my TCP image but now I can't. I wonder if NSLE's developer will notice this... Well I hope so.

**Comment by MayorKitty** [comment]
Any word yet on whether or not this will be updated? The other save editor won't move my buildings and PWPs no matter how many times I try, it's driving me crazy. This one always worked flawlessly for me.

**Comment by Monotek18** [comment]
Late, but I recommend you to use ACSE, it support all AC games:
https://github.com/Cuyler36/ACSE
